.cirrus.yml

```diff
@@ -28,7 +32,7 @@ zkg_ubuntu_task:
     fingerprint_script: echo $CIRRUS_TASK_NAME-$CIRRUS_OS
 
   install_spicy_script:
-    - curl -o spicy-dev.deb https://api.cirrus-ci.com/v1/artifact/github/zeek/spicy/docker_ubuntu20/packages/spicy-dev.deb
+    - curl -o spicy-dev.deb https://api.cirrus-ci.com/v1/artifact/github/zeek/spicy/docker_debian11/packages/spicy-dev.deb
```

Comment:
Aus Neugier: warum der Wechsel zu Debian?

Reply:
I wanted to stay somewhat close to the `Dockerfile` used to publish the Zeek image which uses Debian. Beyond that I don't have a strong preference for what to use here.